### PR TITLE
Release 2020.08.0

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Release Notes 2020.08.0
- **Is this PR related to a Monolith release?** Yes 2020.08.0

### CODE REVIEW

File: packages/@okta/vuepress-site/data/oauth2-error-codes.json [hunk 1]
```diff
@@ -316,6 +316,27 @@
     "internalErrorCode" : "access_token_insufficient_scopes",
     "errorCode" : "insufficient_scope",
     "errorDescription" : "The access token provided does not contain the required scopes."
+  }, {
+    "title" : "Invalid authorization details",
+    "statusCode" : 400,
+    "statusReasonPhrase" : "Bad Request",
+    "internalErrorCode" : "invalid_authorization_details",
+    "errorCode" : "invalid_authorization_details",
+    "errorDescription" : "Invalid authorization_details."
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/data/oauth2-error-codes.json:325]: "Invalid authorization details." (no underscore in the description)

File: packages/@okta/vuepress-site/data/oauth2-error-codes.json [hunk 2]
```diff
@@ -316,6 +316,27 @@
...
+  }, {
+    "title" : "Invalid authorization details with errors",
+    "statusCode" : 400,
+    "statusReasonPhrase" : "Bad Request",
+    "internalErrorCode" : "invalid_authorization_details_with_errors",
+    "errorCode" : "invalid_authorization_details",
+    "errorDescription" : "Invalid authorization_details. Errors: [{0}]."
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/data/oauth2-error-codes.json:332]: "Invalid authorization details. Errors: [{0}]." (no underscore)

File: packages/@okta/vuepress-site/data/oauth2-error-codes.json [hunk 3]
```diff
@@ -316,6 +316,27 @@
...
+  }, {
+    "title" : "Invalid authorization details invalid type",
+    "statusCode" : 400,
+    "statusReasonPhrase" : "Bad Request",
+    "internalErrorCode" : "invalid_authorization_details_invalid_resource_type",
+    "errorCode" : "invalid_authorization_details",
+    "errorDescription" : "The type value {0} in authorization_details is not supported."
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/data/oauth2-error-codes.json:339]: "authorization details" (underscore in description)

File: packages/@okta/vuepress-site/data/oauth2-error-codes.json [hunk 4]
```diff
@@ -862,6 +883,13 @@
     "internalErrorCode" : "authorize_request_object_invalid_signature_algorithm",
     "errorCode" : "invalid_request_object",
     "errorDescription" : "The JWT signature algorithm of the authorize request object is invalid."
+  }, {
+    "title" : "Invalid authorization details request response type",
+    "statusCode" : 400,
+    "statusReasonPhrase" : "Bad Request",
+    "internalErrorCode" : "invalid_response_type",
+    "errorCode" : "invalid_response_type",
+    "errorDescription" : "Invalid rich authorization request. The response_type must be 'code'."
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/data/oauth2-error-codes.json:892]: "response type must be..." (underscore)

File: packages/@okta/vuepress-site/data/oauth2-error-codes.json [hunk 5]
```diff
@@ -1107,6 +1135,13 @@
     "internalErrorCode" : "request_uri_not_supported",
     "errorCode" : "request_uri_not_supported",
     "errorDescription" : "The authorize request uri is not supported by this client."
+  }, {
+    "title" : "Invalid authorization details request scope present",
+    "statusCode" : 400,
+    "statusReasonPhrase" : "Bad Request",
+    "internalErrorCode" : "scope_present",
+    "errorCode" : "scope_present",
+    "errorDescription" : "Invalid rich authorization request. The request should not include 'scope' parameter."
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/data/oauth2-error-codes.json:1144]: should not => shouldn't

File: packages/@okta/vuepress-site/data/oauth2-error-codes.json [hunk 6]
```diff
@@ -1135,6 +1170,13 @@
     "internalErrorCode" : "access_token_not_generated",
     "errorCode" : "server_error",
     "errorDescription" : "Failed to create the access_token."
+  }, {
+    "title" : "Rar token not created",
```
Comment (IanHakes-okta) [inline packages/@okta/vuepress-site/data/oauth2-error-codes.json:1174]: Rar => RAR (is a file type)

File: packages/@okta/vuepress-site/docs/release-notes/2020-08-0/index.md [hunk 7]
```diff
@@ -0,0 +1,25 @@
...
+| [OAuth 2.0 authorization code length has been increased](#oauth-2-0-authorization-code-length-has-been-increased)| August 5, 2020 |
+| [Bugs fixed in 2020.08.0](#bugs-fixed-in-2020-08-0)       | August 5, 2020           |
+
+### Apple as an Identity Provider is now GA in Production
+
+Apple as an Identity Provider is now Generally Available in Production. Apple as an IdP allows users to sign in to your app using their Apple ID. See [Apple as an Identity Provider](/docs/guides/add-an-external-idp/apple/before-you-begin/). <!-- OKTA-302300 -->
+
+### OAuth 2.0 authorization code length has been increased
+
+To better align with [security best practices](https://tools.ietf.org/html/rfc6819#section-5.1.4.2.2 ), the length of OAuth 2.0 authorization codes is now 256 bits of entropy (43 characters). <!-- OKTA-310346 -->
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-08-0/index.md:19]:
```suggestion
To better align with [security best practices](https://tools.ietf.org/html/rfc6819#section-5.1.4.2.2), the length of OAuth 2.0 authorization codes is now 256 bits of entropy (43 characters). <!-- OKTA-310346 -->
```

File: packages/@okta/vuepress-site/docs/release-notes/2020-08-0/index.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+### Bugs fixed in 2020.08.0
+
+* The GET `/api/v1/users/<userid>/idps` and POST `/api/v1/idps/<idpId>/users/<userId>` endpoints weren't [OAuth](/docs/guides/implement-oauth-for-okta/scopes/) enabled. (OKTA-303902)
```
Comment (jakubvul-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2020-08-0/index.md:23]:
```suggestion
* The GET `/api/v1/users/${userid}/idps` and POST `/api/v1/idps/${idpId}/users/${userId}` endpoints weren't [OAuth](/docs/guides/implement-oauth-for-okta/scopes/) enabled. (OKTA-303902)
```

**Pull request merged**